src/main/java/com/synopsys/integration/detect/Application.java

```diff
@@ -208,6 +208,12 @@ private void createStatusOutputFile(FormattedOutputManager formattedOutputManage
             Gson formattedGson = new GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create();
             String json = formattedGson.toJson(formattedOutputManager.createFormattedOutput(detectInfo));
             FileUtils.writeStringToFile(statusFile, json, Charset.defaultCharset());
+            
+            if (directoryManager.getJsonStatusOutputDirectory() != null) {
+                File statusCopyFile = new File(directoryManager.getJsonStatusOutputDirectory(), "status.json");
+                logger.info("Creating copy of status file: {}", statusCopyFile);
```

Comment:
We could avoid hard coding "status.json" twice by either using statusFile.getName() on line 213, or creating a constant.